persistence/src/main/java/de/starwit/persistence/entity/Attribute.java

```diff
@@ -40,6 +41,14 @@
 	@Column(name = "NULLABLE")
 	private boolean nullable = true;
 
+	//TODO
```

Comment:
TODO what? 
Is something not finished there?

Reply:
Yes - unique and patternJava is not implemented right now